src/lib/example.js

```diff
@@ -135,15 +135,32 @@ async function setProjectName(projDir) {
     'PROJECT_NAME',
     titleCase(name)
   );
+
   replaceInFile(
     path.join(projDir, 'package.json'),
     'package-name',
     kebabCase(name)
   );
 
+  addStartScript(path.join(projDir, 'package.json'));
+
   spin.succeed(_green(step));
 }
 
+/**
+ * Helper to add start script to package.json.
+ * @param {string} file    Path to file
+ * @param {string} script  Start script to add.
+ */
+function addStartScript(file) {
+  const startScript = 'node build/src/index.js';
```

Comment:
Would probably simplify to not use a var. Should be `'node build/src/run.js'` now right?